setup.cfg

```diff
@@ -1,13 +1,16 @@
 [metadata]
 name = quickmpc
-version = 0.1
 author = Acompany
+license=Apache 2.0 Licences
+license_file=LICENSE
 
 [options]
-include_package_data = True
-packages = find:
+setup_requires=setuptools>=42; setuptools_scm[toml]>=3.4
```

Comment:
In [pyproject.toml](https://github.com/acompany-develop/QuickMPC-libClient-py/pull/15/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711), It is written`["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.2"]`
Is it OK if the VERSION is different?

Reply:
Thx!